octopus_deploy/datadog_checks/octopus_deploy/constants.py

```diff
@@ -13,11 +13,14 @@
 DEPLOY_SUCCESS_METRIC = f"{DEPLOY_PREFIX}.succeeded"
 DEPLOY_RERUN_METRIC = f"{DEPLOY_PREFIX}.can_rerun"
 DEPLOY_WARNINGS_METRIC = f"{DEPLOY_PREFIX}.has_warnings_or_errors"
+DEPLOY_QUEUED_METRIC = f"{DEPLOY_PREFIX}.queued"
```

Comment:
Are these two metrics `queued` and `running` necessary since we're already reporting the state in a tag `task_state`?